src/puppetlabs/trapperkeeper/authorization/ring.clj

```diff
@@ -2,6 +2,10 @@
   (:require [schema.core :as schema])
   (:import (java.security.cert X509Certificate)))
 
+(def cert-key
+  "The nested key where the client certificate for the request is stored"
+  [:authorization :certificate])
+
 (def is-authentic-key
   "The nested key where authenticity information is stored."
   [:authorization :authentic?])
```

Comment:
Ok, I'll look into doing the get/set functions instead.  For cases where there are error messages referencing the name of the header, suppose I'll just define them explicitly at the point they are referenced rather than referring to a constant.